<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterialWindow</class>
 <widget class="QDialog" name="AddMaterialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Material</string>
  </property>
  <layout class="QVBoxLayout" name="addMaterialVerticalLayout">
   <item>
    <widget class="QLabel" name="materialTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add New Material</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="addMaterialFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="materialLabel">
       <property name="text">
        <string>Material</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="materialInput">
       <item>
        <property name="text">
         <string>Rubber Screen Cloth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tacky Back</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polyurethane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moulded Polyurethane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rubber x60</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hardnessLabel">
       <property name="text">
        <string>Hardness</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="thicknessLabel">
       <property name="text">
        <string>Thickness</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="hardnessInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="thicknessInput">
       <property name="suffix">
        <string>mm</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtons</sender>
   <signal>accepted()</signal>
   <receiver>AddMaterialWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtons</sender>
   <signal>rejected()</signal>
   <receiver>AddMaterialWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
